inference-engine/tests/functional/shared_test_classes/include/shared_test_classes/base/layer_test_utils.hpp

```diff
@@ -144,6 +145,11 @@ class LayerTestsCommon : public CommonTestUtils::TestsCommon {
     float threshold;
     InferenceEngine::CNNNetwork cnnNetwork;
     std::shared_ptr<InferenceEngine::Core> core;
+    ngraph::PartialShape inputDynamicShape;
+    ngraph::Shape targetStaticShape;
+    std::vector<ngraph::Shape> targetStaticShapes;
```

Comment:
Looks a bit confusing: 
```
    ngraph::Shape targetStaticShape;
    std::vector<ngraph::Shape> targetStaticShapes;
```

I mean that  `ngraph::Shape targetStaticShape;` looks a special case of `std::vector<ngraph::Shape> targetStaticShapes;`